assets/css/styles.css

```diff
@@ -0,0 +1,165 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  background-color: #fff4e3;
+  color: #816068;
+  display: flex;
+  flex-direction: column;
+  font-family: omnes-pro, Arial, Helvetica, sans-serif;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.no-shrink {
+  flex-shrink: 0;
+}
+
+.background {
+  z-index: -1;
```

Comment:
I see what this is trying to do: prevent the wave from blocking clicks on your social links. But z-index is generally a code smell, and I'd argue it's better to accomplish this through HTML element order.

```
<top-wave />
<bottom-wave />
<main />
```

You can position the waves with CSS, but `<main />` is always "in front", i.e. on top of the stacking context.

Reply:
Discussed offline :)